File: src/numerical/fourFifthsProblem/test_four_fifths_problem.py
from four_fifths_problem import *
from time import time

start = time()
result = four_fifths_problem(150)
end = time()
elapsed = int((end - start) * 1000)
print(f'Время выполнения {elapsed}ms')
print(result)

'Время выполнения 16183ms'
[[27, 84, 110, 133, 144]]

File: src/numerical/fourFifthsProblem/TestFourFifthsProblem.java
package numerical.fourFifthsProblem;

import static numerical.fourFifthsProblem.FourFifthsProblem.fourFifthsProblem;

public class TestFourFifthsProblem {
    public static void main(String[] args) {
        long start = System.currentTimeMillis();
        System.out.println(fourFifthsProblem(150));
        long end = System.currentTimeMillis();
        System.out.printf("Время выполнения - %dms", end - start);
    }   
}

